### PR TITLE
Iy rds passwd

### PR DESCRIPTION
**Pull request by iiman**
## Approvals

- [x] Verified
- [x] @TimMoore
- [ ] @hebbo
- [ ] @ptolemy428
- [x] @iiman

**NB** Actually we don't need this PR all together after merging [this PR](https://github.com/cultureamp/docker-ansible/pull/13). This branch will temporarily serve as a connector between the new docker-ansible and the cloudformation-stacks stacks.

rds_password arg added as one of valid cmd args.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM cultureamp/docker-ansible:latest
+FROM cultureamp/docker-ansible:iy_rds_passwd
 MAINTAINER Cultureamp IS Team <is_team@cultureamp.com>
```
Comment (iiman) [inline Dockerfile]: This will be changed to `latest` after merging [this PR](https://github.com/cultureamp/docker-ansible/pull/13)

**Comment by iiman** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;lXaFbeaM&#x2F;511-create-rds-stack-with-password-passed-in-as-cfn-parameters"> Create rds stack with password passed in as cfn parameters.</a>

File: cloudformationcore/utils/validator.py [hunk 2]
```diff
@@ -7,6 +7,7 @@ def validate_args():
     parser.add_argument('-region', required=True, type=str)
     parser.add_argument('-stack_prefix', required=True, type=str)
     parser.add_argument('-config', required=False, type=str)
+    parser.add_argument('-rds_password', required=False, type=str)
```
Comment (ptolemy428) [inline cloudformationcore/utils/validator.py]: We shouldn't need this change given it's a 'launch time only' thing?

Comment (iiman) [inline cloudformationcore/utils/validator.py]: That is correct. Actually we don't need this PR all together after merging [this PR](https://github.com/cultureamp/docker-ansible/pull/13)

**Pull request closed without merging**